science/iverilog/Portfile

```diff
@@ -1,20 +1,21 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-
 PortSystem          1.0
 PortGroup           github 1.0
+PortGroup           compiler_blacklist_versions 1.0
+
+name                iverilog
+github.setup        steveicarus iverilog 20250103 s
+github.tarball_from archive
 
-github.setup        steveicarus iverilog 12_0 v
-# Change github.tarball_from to 'releases' or 'archive' next update
-github.tarball_from tarball
-version             [string map {_ .} ${github.version}]
+version             20250103
```

Comment:
This explicit `version` line can probably be deleted because it is being implicitly set by `github.setup`